brightlayer-ui/doc-it
# Add latest list patterns to patterns doc

#### Changes proposed in this Pull Request:
- Add latest list patterns to patterns doc


#### Screenshots:

![image](https://user-images.githubusercontent.com/8997218/149804765-fe8b2c7c-2ef8-4c41-88df-f56b2b9d3718.png)


#### To Test:
- Go to http://localhost:3000/patterns/lists
- Verify that all the code example CTA links work
